src/tools/dev/scripts/bv_support/bv_openvkl.sh

```diff
@@ -0,0 +1,177 @@
+function bv_openvkl_initialize
```

Comment:
Another new module? Sep PR or maybe combine with the others here.

Reply:
Actually that one may not be need for OSPRay but it slipped in.